### DISCUSSION TITLE
Hide my repository

### DISCUSSION THREAD

**Discussion by Micheljd12** [community/community#39789] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello , how can i hide my code repositery ?

**Comment by mouradcherif** [comment, marked as answer]
Go to your Repository  > Settings > Danger Zone > on Change repository visibility click Change visibility then click change to private.
Now your repository and its code will be visible only to you.

**Reply by Micheljd12** [reply]
Thank you !
